aptos-move/aptos-vm/tests/sharded_block_executor.rs

```diff
@@ -35,6 +36,7 @@ fn test_partitioner_v2_uniform_sharded_block_executor_no_conflict() {
 }
 
 #[test]
+#[ignore]
```

Comment:
why are these ignored?

Reply:
they don't support fa now. will let @manudhundi to fix it later.